### PR TITLE
fix(loader): build the loader before publish

### PR DESCRIPTION
**Pull request by EoleO**
### Bug Fixes

- build the loader before publish

### Chores

- update `schema-utils`

### CODE REVIEW

**Review by apascal — APPROVED** [review]

**Review by apascal — COMMENTED** [review]
Rather build the distributed file on your side with a task explained in the readme to do it

File: .husky/pre-commit [hunk 1]
```diff
@@ -3,3 +3,4 @@
 
 npx --no-install tsc --project "tsconfig.development.json"
 npx --no-install lint-staged
+npm run build & git add dist/
```
Comment (apascal) [inline .husky/pre-commit]: This is not linked to pre-commit, but rather prepublish, I would remove it all together as this is your responsibility to rebuild before pushing a new PR

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,16 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [2.0.1](https://github.com/LesEchos/image-size-loader/compare/v2.0.1...v2.0.0) (2021-07-12)
+
+### Bug Fixes
+
+- precommit build the loader
```
Comment (apascal) [inline CHANGELOG.md]: unclear

**Review by apascal — CHANGES_REQUESTED** [review]

File: .husky/pre-commit (hunk 1 above)
Comment (EoleO) [reply .husky/pre-commit]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (EoleO) [reply CHANGELOG.md]: Done

**Review by apascal — APPROVED** [review]

**Pull request merged**